src/components/tools/Charts/LineArea/Chart.svelte

```diff
@@ -60,27 +65,52 @@
 
   $: if (Array.isArray(data) && !isEmptyData(data)) {
```

Comment:
Just wondering if this should be an `else` after the first `if` or maybe a nested `if / else`:

```js
$: if (Array.isArray(data)) {
  if (isEmptyData(data) {
    // handle empty data
  } else {
    // set chart props
  }
}
```
But that might mess with the reactive statements, not sure. Alternatively the `isEmptyData` function could check to see if the `data` array is an array.

Reply:
Moved the check if `data` is array into the helper function in https://github.com/berkeley-gif/caladapt-website-2021/pull/114/commits/4de29c3ff081545cbd9336d1bc4e8f09de34b036